docs/output.md

```diff
@@ -354,37 +357,44 @@ An example MultiQC report generated from a full-sized dataset can be viewed on t
-- [Variant calling](#illumina-variant-calling)
-  - [Bowtie 2](#bowtie-2) - Read alignment relative to reference genome
-  - [SAMtools](#samtools) - Sort, index and generate metrics for alignments
-  - [iVar trim](#ivar-trim) - Primer sequence removal for amplicon data
-  - [picard MarkDuplicates](#picard-markduplicates) - Duplicate read marking and removal
-  - [picard CollectMultipleMetrics](#picard-collectmultiplemetrics) - Alignment metrics
-  - [mosdepth](#mosdepth) - Whole-genome and amplicon coverage metrics
-  - [iVar variants](#ivar-variants) _||_ [BCFTools call](#bcftools-call) - Variant calling
-    - [SnpEff and SnpSift](#snpeff-and-snpsift) - Genetic variant annotation and functional effect prediction
-    - [Freyja](#freyja) - Relative lineage abundance analysis from mixed SARS-CoV-2 samples (typically wastewater)
-    - [ASCIIGenome](#asciigenome) - Individual variant screenshots with annotation tracks
-  - [iVar consensus](#ivar-consensus) _||_ [BCFTools and BEDTools](#bcftools-and-bedtools) - Consensus sequence generation
-    - [QUAST](#quast) - Consensus assessment report
-    - [Pangolin](#pangolin) - Lineage analysis
-    - [Nextclade](#nextclade) - Clade assignment, mutation calling and sequence quality checks
-  - [Variants long table](#variants-long-table) - Collate per-sample information for individual variants, functional effect prediction and lineage analysis
-- [De novo assembly](#illumina-de-novo-assembly)
-  - [Cutadapt](#cutadapt) - Primer trimming for amplicon data
-  - [SPAdes](#spades) _||_ [Unicycler](#unicycler) _||_ [minia](#minia) - Viral genome assembly
-    - [BLAST](#blast) - Blast to reference assembly
-    - [ABACAS](#abacas) - Order contigs according to reference genome
-    - [PlasmidID](#plasmidid) - Assembly report and visualisation
-    - [Assembly QUAST](#assembly-quast) - Assembly quality assessment
-- [Workflow reporting and genomes](#illumina-workflow-reporting-and-genomes)
-  - [MultiQC](#multiqc) - Present QC for raw reads, alignment, assembly and variant calling
-  - [Reference genome files](#reference-genome-files) - Save reference genome indices/files
+- [Introduction](#introduction)
+- [Illumina: Pipeline overview](#illumina-pipeline-overview)
+  - [Illumina: Preprocessing](#illumina-preprocessing)
+    - [cat](#cat)
+    - [FastQC](#fastqc)
+    - [fastp](#fastp)
+    - [Kraken 2](#kraken-2)
+  - [Illumina: Variant calling](#illumina-variant-calling)
+    - [Bowtie 2](#bowtie-2)
+    - [SAMtools](#samtools)
+    - [iVar trim](#ivar-trim)
+    - [picard MarkDuplicates](#picard-markduplicates)
+    - [picard CollectMultipleMetrics](#picard-collectmultiplemetrics)
+    - [mosdepth](#mosdepth)
+    - [iVar variants](#ivar-variants)
+    - [BCFTools call](#bcftools-call)
+    - [SnpEff and SnpSift](#snpeff-and-snpsift)
+    - [Freyja](#freyja)
+    - [ASCIIGenome](#asciigenome)
+    - [iVar consensus](#ivar-consensus)
+    - [BCFTools and BEDTools](#bcftools-and-bedtools)
```

Comment:
is this toc really still needed?

Reply:
I kept it as it was there, I'm just doing the TEMPLATE sync here, I'll check docs in a separate PR